texas-hearing-institute/pages/SpeechSettings/tabs/FinalConsonants.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import { ScrollView, View, Text, StyleSheet } from 'react-native';
+import SoundGrid from '../components/SoundGrid';
+import RadioButton from '../../../components/RadioButton';
+import FloatingButton from '../../../components/FloatingButton';
+
+export default function FinalConsonants() {
+	// TODO: pull from sound inventory (need to change sound inventory data structure)
+	const sounds = [
```

Comment:
these three tab files can be consolidated into one component -> just need to change the sounds (which will get pulled from the sound inventory), title, and subtitle based on some param. i think this should wait until after we figure out how we're arranging the updated sound inventory though

Reply:
makes sense, i think we can just leave as is for now